org.gnome.Calendar.json

```diff
@@ -117,8 +117,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz",
-                    "sha256": "c3ee8728e4364b0397f435fa20f92f901ab139d2b264f4e059d67b3c0f43cd36",
+                    "url": "https://download.gnome.org/sources/gcr/4.3/gcr-4.3.90.3.tar.xz",
```

Comment:
I think this is also an unstable release

Reply:
Unfortunately, this won't be covered by x-checker-data supporting .90 releases seems a bit far fetched.